docs/src/architecture/hardening.md

```diff
@@ -68,3 +68,13 @@ The host kernel runs on bare metal. The kernel is provided either with Linux ups
 The host kernel hardening is based on Linux `make tinyconfig`. The default `tinyconfig` fails to assertions on NixOS without modifications. Assertions are fixed in `ghaf_host_hardened_baseline` Linux configuration under Ghaf `modules/host/`.
```

Comment:
in **the** `ghaf_host_hardened_baseline` configuration

Reply:
Updated as per review comment.